pkg/operator/ceph/nfs/security_test.go

```diff
@@ -51,7 +51,7 @@ func mockReconcile() *ReconcileCephNFS {
 		},
 		cephClusterSpec: &cephv1.ClusterSpec{
 			CephVersion: cephv1.CephVersionSpec{
-				Image: "quay.io/ceph/ceph:v17.2.4",
```

Comment:
@BlaineEXE Does this test need the specific tag, or could we just use the `v17` tag so we don't update it with every ceph patch?

Reply:
`v17` is fine